connector/src/main/java/org/geysermc/connector/entity/player/PlayerEntity.java

```diff
@@ -223,6 +223,16 @@ public void setPosition(Vector3f position) {
         this.position = position.add(0, entityType.getOffset(), 0);
     }
 
+    /**
+     * Set the player position without adding the entity type's offset. Used when the player sends us a move packet
+     * where the offset is already added
+     *
+     * @param position the new position of the Bedrock player
+     */
+    public void setPositionWithoutOffset(Vector3f position) {
+        this.position = position;
+    }
+
```

Comment:
This method should be removed and instead updated with:
```suggestion
    public void setPosition(Vector3f position, boolean includeOffset) {
        this.position = includeOffset ? position.add(0, entityType.getOffset(), 0) : position;
    }
```
Then just have the current `setPosition` overload this method.